dpdk.spec

```diff
@@ -297,28 +271,33 @@ sed -i -e 's:-%{machine_tmpl}-:-%{machine}-:g' %{buildroot}/%{_sysconfdir}/profi
 %{incdir}/
 %{sdkdir}/
 %if %{with tools}
-%exclude %{sdkdir}/usertools/
 %endif
 %if %{with examples}
 %exclude %{sdkdir}/examples/
 %endif
 %{_sysconfdir}/profile.d/dpdk-sdk-*.*
 %{_libdir}/*.so
+%{pmddir}/*.so
```

Comment:
Surely these are unversioned DSOs that are needed at runtime, much like daos has in /usr/lib64/daos_srv/